paymentsheet/src/main/java/com/stripe/android/link/ui/inline/LinkInlineSignupFields.kt

```diff
@@ -160,4 +177,56 @@ internal fun LinkInlineSignupFields(
     }
 }
 
+@Composable
+private fun ErrorText(
+    text: String,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Icon(
+            modifier = Modifier
+                .padding(horizontal = 10.dp, vertical = 12.dp)
+                .size(20.dp),
+            painter = painterResource(id = R.drawable.stripe_ic_sail_warning_circle),
+            contentDescription = null,
+            tint = MaterialTheme.colors.error
+        )
+        HyperlinkedText(
+            text = text,
+            modifier = Modifier
+                .padding(vertical = 12.dp),
+            color = MaterialTheme.colors.error,
+            style = TextStyle(
+                fontFamily = FontFamily.Default,
+                fontWeight = FontWeight.Normal,
+                fontSize = 14.sp,
+                lineHeight = 20.sp
+            )
+        )
+    }
+}
```

Comment:
separated Inline signup error view from the Link one, so that we can update the Link flow designs without impacting  the inline view.